sdk/storage/Azure.Storage.Files.Shares/src/ShareDirectoryClient.cs

```diff
@@ -1314,25 +1320,31 @@ private void SetNameFieldsIfNull()
         /// notifications that the operation should be cancelled.
         /// </param>
         /// <returns>
-        /// A <see cref="Response"/> describing the status of the
-        /// <see cref="ForceCloseHandle"/> operation.
+        /// A <see cref="ClosedHandlesInfo"/> describing the status of the
+        /// <see cref="ForceCloseHandleAsync"/> operation.
         /// </returns>
         /// <remarks>
         /// A <see cref="RequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
-        public virtual async Task<Response> ForceCloseHandleAsync(
+        public virtual async Task<ClosedHandlesInfo> ForceCloseHandleAsync(
```

Comment:
Should this be `Response<ClosedHandlesInfo>`?

Reply:
Sure, I can change it.  What do you think of Krzysztof's suggestion to rename ClosedHandlesInfo -> ClosedHandlesResult?

Reply:
I think it makes sense and makes the name a little more friendly for future additions.

Reply:
BTW, I suggested CloseHandlesResult, not ClosedHandlesResult.